### PR TITLE
Add RestoreDefaultVideoColor implementation

### PR DESCRIPTION
**Pull request by XiaosongWei**
Restore the default values for VPP and then remove the effect applied
before.

Jira: None
Test: Set/Restore color works as expected

Change-Id: I8e86d0a6fbb6723e661a5c14d7470cfd97ac18ea
Tracked-On:
Signed-off-by: Xiaosong Wei <xiaosong.wei@intel.com>

### CODE REVIEW

**Comment by XiaosongWei** [comment]
@kalyankondapally @feiwan1 PTAL

**Comment by kalyankondapally** [comment]
@XiaosongWei We have few patches waiting to be merged to master based on test resutls:
https://github.com/intel/IA-Hardware-Composer/tree/scalar, Can u rebase this across that branch as Va backend has changed here. Thanks.

**Comment by XiaosongWei** [comment]
@kalyankondapally

File: common/compositor/va/varenderer.cpp [hunk 1]
```diff
@@ -305,10 +315,17 @@ bool VARenderer::UpdateCaps() {
     VAProcFilterParameterBufferColorBalance cbparam;
     VAProcFilterParameterBuffer sharpparam;
 
-    for (ColorBalanceCapMapItr itr = colorbalance_caps_.begin();
-         itr != colorbalance_caps_.end(); itr++) {
+    for (auto itr = colorbalance_caps_.begin(); itr != colorbalance_caps_.end();
+         itr++) {
+      bool use_default =
+          itr->second.use_default_ &&
+          itr->second.value_ != itr->second.caps_.range.default_value;
```
Comment (kalyankondapally) [inline common/compositor/va/varenderer.cpp]: shouldn't this be "=="

Comment (XiaosongWei) [reply common/compositor/va/varenderer.cpp]: no. after restoring the default value, filter is not required any more.

**Comment by kalyankondapally** [comment]
merged

**Pull request closed without merging**